wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -13,6 +13,7 @@
     <script src="{% versioned_static 'wagtailadmin/js/vendor/modernizr-2.6.2.min.js' %}"></script>
 
     <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/normalize.css' %}" />
+    <link rel="preload" href="{% url 'wagtailadmin_sprite' %}">
```

Comment:
@thibaudcolas 

- This prevents a flash when in development the sprite isn't cached. 
- It also removes the initial flash.


But we still need